src/Compilers/CSharp/Test/Syntax/LexicalAndXml/LexicalTests.cs

```diff
@@ -2616,6 +2620,52 @@ public void TestNumericWithUnderscoresWithoutFeatureFlag()
 
         [Fact]
         [Trait("Feature", "Literals")]
+        public void TestNumericWithLeadingUnderscores()
+        {
+            var text = "0x_A";
```

Comment:
Also test for trailing underscore: "0x_"

---
In reply to: [131779530](https://github.com/dotnet/roslyn/pull/20449#discussion_r131779530) [](ancestors = 131779530)